server/utils/sessionStatus.ts

```diff
@@ -0,0 +1,28 @@
+import { ActionPlanAppointment } from '../services/interventionsService'
+
+export enum SessionStatus {
+  notScheduled,
+  scheduled,
+  completed,
+  didNotAttend,
+}
+
+export default {
+  forAppointment: (appointment: ActionPlanAppointment): SessionStatus => {
```

Comment:
I think it would be good to test this - it's a self-contained bit of logic that will probably only get more complex as the service evolves. Perhaps worth a separate commit too.

Reply:
Have added a test for this and split into its own commit.